concepts/05 Widgets/Slider/05 Customize Widget Appearance.md

```diff
@@ -31,8 +129,106 @@ The **Slider** can also display a tooltip for the slider handle. To configure it
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-slider
+        <dxo-tooltip
+            [enabled]="true"
+            [format]="format"
+            showMode="always"
+            position="bottom"> <!-- or "top" -->
+        </dxo-tooltip>
+    </dx-slider>
+
+    <!--TypeScript-->
+    import { DxSliderModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        format = function(value) {
+            return value + " units";
+        };
+    }
+    @NgModule({
+        imports: [
+            // ...
+            DxSliderModule
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <template>
+        <DxSlider>
+            <DxTooltip
+                :enabled="true"
+                position="bottom"
+                showMode="always"
+                :format="format"
+            />
+        </DxSlider>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxSlider, DxTooltip } from 'devextreme-vue/slider';
+
+    export default {
+        components: {
+            DxSlider,
+            DxTooltip
+        },
+        methods: {
+            format(value) {
+                return value + " units";
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Slider, Tooltip } from 'devextreme-react/slider';
+
+    const format = function(value) {
+        return value + " units";
+    }
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+        }
+
+        render() {
+            return (
+                <Slider>
+                    <Tooltip
+                        enabled={true}
+                        position="bottom" // or "top"
+                        showMode="always" // or "onHover"
```

Comment:
I see two comments here that suggest different options. Why did you add them here but skipped them in Angular and Vue?

Reply:
fixed